generators/server/files.js

```diff
@@ -61,12 +61,15 @@ const serverFiles = {
     ],
     serverResource: [
         {
+            path: SERVER_MAIN_RES_DIR,
+            templates: [{ file: 'default_banner.txt', method: 'copy', noEjs: true }, 'application.properties']
+        },
+        {
+            condition: generator => generator.authenticationType === 'jwt',
             path: SERVER_MAIN_RES_DIR,
             templates: [
-                { file: 'default_banner.txt', method: 'copy', noEjs: true },
                 { file: 'jwt/privateKey.pem', method: 'copy', noEjs: true },
                 { file: 'META-INF/resources/publicKey.pem', method: 'copy', noEjs: true },
```

Comment:
We don't need public/private keys for Spring Security. Are you sure it's necessary here? [This blog post shows Quarkus + OIDC with Okta](https://developer.okta.com/blog/2020/04/08/kafka-streams) and I don't see any keys (except to secure Kafka).

Reply:
Are you sure you're looking the version of this file?
https://github.com/jhipster/generator-jhipster-quarkus/blob/d9bd816508f607d5c0b838821d94de8ad5d45fa5/generators/server/files.js#L68
I've added a condition to generate the keys only when jwt is choosen